demos/datepicker/datepicker-fr.js

```diff
@@ -6,7 +6,7 @@
 	if ( typeof define === "function" && define.amd ) {
 
 		// AMD. Register as an anonymous module.
-		define([ "../jquery.ui.datepicker" ], factory );
+		define([ "datepicker" ], factory );
```

Comment:
Well, regardless, this file can't be modified directly. It's currently out-of-date, but this needs to be an exact copy of the source file.